pkg/model/const.go

```diff
@@ -317,4 +317,7 @@ const (
 
 	// OktetoDefaultImageTag default tag assigned to image to build
 	OktetoDefaultImageTag = "okteto"
+
+	// OktetoImageTagWithVolumes is the tag assigned to an image with volume mounts
+	OktetoImageTagWithVolumes = "okteto-with-volume-mounts"
```

Comment:
can we build the same tag in both cases? I think it would simplify the code

Reply:
I don't think it's a good idea because of this use case:
1.  User run `okteto deploy` for the first time 
2. It builds the first image with the tag `okteto`
3. It fails on including the volumes
4. User runs again `okteto deploy` and it won't be built because the tag exists (right now we are checking if the tag with the volumes exists)

Reply:
Makes sense